apps/system/js/stack_manager.js

```diff
@@ -103,6 +103,7 @@ var StackManager = {
     // queueing.
     if (this._didntMove) {
       window.dispatchEvent(new CustomEvent('sheets-gesture-end'));
+      this._appOut && this._appOut._setActive(true);
```

Comment:
if we're going with that we should expose a clean `setActive` method (without the `_`)